hedy.py

```diff
@@ -1687,6 +1697,16 @@ def __init__(self, lookup, numerals_language):
         super().__init__(lookup, numerals_language)
         self.ifpressed_prefix_added = False
 
+    def list_access_var(self, meta, args):
+        var = escape_var(args[0])
+         # if we tried to use 'at random' in level 16 raise an exception
```

Comment:
I don't really understand this command because I don't see what exception you are referring to here?